lib/crypto/curve.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Tezex.Crypto.Curve do
+  @moduledoc false
+
+  @doc """
+  Specific elliptic curve data.
+
+  Parameters:
+    - `:A` [number]: angular coefficient of x in the curve equation. ex: 123
+    - `:B` [number]: linear coefficient of x in the curve equation. ex: 123
+    - `:P` [number]: curve modulo. ex: 12345
+    - `:N` [number]: curve order. ex: 12345
+    - `:G` [%Point]: EC Point corresponding to the public key. ex: %Point{x: 123, y: 456}
+    - `:name` [string]: curve name. ex: "secp256k1"
```

Comment:
`name` is an atom when it's used later on `%Curve{name: :secp256k1}`, could we had a `@type t()` to this structure?

Reply:
Good point 👍